### PR TITLE
rcouto/logrus

### PR DESCRIPTION
**Pull request by ryancouto**
This PR is the alternative to https://github.com/scootdev/scoot/pull/230, using logrus instead of log15. There is no structured logging included or modifications to logged statements, just import path changes and fmt replacements.

### CODE REVIEW

File: binaries/apiserver/main.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func main() {
 	configFlag := flag.String("config", "{}", "API Server Config (either a filename like local.local or JSON text")
 	flag.Parse()
 
-	log.SetFlags(log.LstdFlags | log.LUTC | log.Lshortfile)
+	// log.SetFlags(log.LstdFlags | log.LUTC | log.Lshortfile)
```
Comment (dgassaway) [inline binaries/apiserver/main.go]: Prefer deletion, or better: setup with Logrus' SetFormatter(TextFormatter), SetLevel, etc

Comment (jasonbs10) [inline binaries/apiserver/main.go]: We should definitely continue to include file:Line# in this PR.

File: saga/saga_generators.go [hunk 2]
```diff
@@ -39,7 +40,7 @@ func genSagaState(genParams *gopter.GenParameters, includeJob bool) *SagaState {
 	state, err := makeSagaState(sagaId, job)
 
 	if err != nil {
-		fmt.Println(err)
+		log.Println(err)
```
Comment (JeanetteBruno) [inline saga/saga_generators.go]: change Print to Info?

File: binaries/apiserver/main.go (hunk 1 above)
Comment (JeanetteBruno) [inline binaries/apiserver/main.go]: +1 Using SetFormatter(TextFormatter)

File: async/Mailbox_test.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ func storeValue_withMailbox(num int) error {
 			successfulWrites++
 		}
 		returnedWrites++
-		fmt.Println("completedWrites", returnedWrites)
+		log.Println("completedWrites", returnedWrites)
```
Comment (JeanetteBruno) [inline async/Mailbox_test.go]: do we want to convert all log.Print to log.Info?  (I also saw one in sagaGenerators... not sure if there are more lurking out there.

File: binaries/apiserver/main.go (hunk 1 above)
Comment (ryancouto) [inline binaries/apiserver/main.go]: Should've said this in the summary, but I was planning to include file:Line# in the subsequent Twitter source phab, I think it makes sense to leave OSS as barebones as possible and do all custom configuration within source.

File: async/Mailbox_test.go (hunk 3 above)
Comment (ryancouto) [inline async/Mailbox_test.go]: Yes, thank you.

**Pull request merged**